packages/suite/src/components/suite/AccountTypeBadge.tsx

```diff
@@ -0,0 +1,26 @@
+import { AccountType, UppercaseAccountType } from '@suite-common/wallet-types';
+import { toUppercaseType } from '@suite-common/suite-utils';
+import { Badge, BadgeSize } from '@trezor/components';
+import { Translation } from './Translation';
+
+type AccountTypeBadgeProps = {
+    accountType?: AccountType;
+    size?: BadgeSize;
+};
+
+export const AccountTypeBadge: React.FC<AccountTypeBadgeProps> = ({
```

Comment:
Is there a reason why to use React.FC here? We almost don't use it throughout our code and I [saw that](https://backstage.io/docs/architecture-decisions/adrs-adr006/) it's not even recommended anymore.

If there is a reason please import it directly not via React global object.